repository/channel_impl.go

```diff
@@ -439,6 +439,57 @@ func (repo *GormRepository) GetChannelStats(channelID uuid.UUID) (*ChannelStats,
 	}
 
 	var stats ChannelStats
+	if err := repo.db.Unscoped().
+		Model(&model.Message{}).
+		Select("COUNT(channel_id) As total_message_count").
+		Where(&model.Message{ChannelID: channelID}).
+		Find(&stats.TotalMessageCount).
+		Error; err != nil {
+		return nil, err
+	}
+	var allStampCount []struct {
+		StampID    uuid.UUID
+		StampCount int64
+		TotalCount int64
+	}
+
+	if err := repo.db.
+		Unscoped().
+		Model(&model.Message{}).
+		Select("stamp_id As stamp_id", "COUNT(stamp_id) As stamp_count", "SUM(count) As total_count").
+		Joins("messages join messages_stamps on messages.id=messages_stamps.message_id").
```

Comment:
最初の`messages`は抜いて良さそう（これでも動いてるのかな？）
```suggestion
		Joins("JOIN messages_stamps ON messages.id = messages_stamps.message_id").
```